runtime/connectors/local_file/file.go

```diff
@@ -1,4 +1,4 @@
-package file
```

Comment:
Call the folder and package `localfile` (see https://google.github.io/styleguide/go/decisions#package-names). We can keep the constant `"local_file"`

Reply:
Updated